conf/config-default.yaml

```diff
@@ -208,3 +209,9 @@ plugin_attr:
   log-rotate:
     interval: 3600    # rotate interval (unit: second)
     max_kept: 168     # max number of log files will be kept
+#  error-log-logger:
```

Comment:
we can set those by `plugin_metadata` via Admin API.

https://github.com/apache/apisix/blob/master/apisix/plugins/http-logger.lua#L58-L71